trending_deploy/deploy.py

```diff
@@ -57,35 +57,66 @@ def deploy_model(model: Model) -> bool:
     """
     try:
         model_name = model.model_info.id
-        endpoint_name = f"{ENDPOINT_PREFIX}{model_name.replace('/', '-')}"
+        endpoint_name = f"{ENDPOINT_PREFIX}{model_name.replace('/', '-')}"[:30].lower()
```

Comment:
Do you know if it's okay for 2 endpoints to have the same name? If not, the `[:30]` is a bit risky

Reply:
Looks like collisions will be extremely common like this
![image](https://github.com/user-attachments/assets/0d4f0279-01f6-45bf-9a38-dbd10184b94d)
![image](https://github.com/user-attachments/assets/3c6ac5c4-6329-4d1c-aefc-8da426d2a367)

Reply:
got an idea